src/backend/llvm_ast.c

```diff
@@ -432,6 +432,14 @@ struct LLVMOpaqueModule *blvm_create_module(
     if (compiler_args_print_backend_debug(ctx->args)) {
         bllvm_mod_debug(ctx->llvm_mod, mod_name);
     }
+
+    size_t mod_id_len;
+    char *mod_id = LLVMGetModuleIdentifier(ctx->llvm_mod, &mod_id_len);
```

Comment:
This seems to be available only on the latest LLVM version (3.9). I do have it locally for example, but Travis tests use use LLVM version 3.7. I would like to be able to compile Refu with 3.7 too so perhaps use the module name that is already known and exists in this same function?

It should be the cstring `mod_name` found here: https://github.com/refu-lang/refu/pull/40/files#diff-10b12b4c46f372efd867280d766ffeadL375

If you do so then you should also move `RFS_POP()` to the end of the function, both in the happy case and the fail case.